lib/internal/test_runner/test.js

```diff
@@ -288,6 +288,13 @@ class Test extends AsyncResource {
     };
     this.waitingOn = 0;
     this.finished = false;
+
+    if (!testOnlyFlag && (only || this.runOnlySubtests)) {
+      const warning =
+        "'only' does not work when node is started" +
+        ' without the --test-only command-line option';
```

Comment:
This wording is a double negative. Please reword this (and add a period to the end of the sentence).

Reply:
Pushed an update. Happy to take any suggestions on the warning message.